Cargo.toml

```diff
@@ -13,6 +13,7 @@ edition = "2018"
 
 [dependencies]
 atomicwrites = "0.2.3"
+ctrlc = { version = "3.1.6", features = ["termination"] }
 crossbeam-channel = "0.3.8"
```

Comment:
"cr[ossbeam-channel]" comes before "ct[rlc]" in the alphabet, would you mind moving the new dependency down by one?

Reply:
Thanks! Just fixed this.